README.md

```diff
@@ -14,3 +14,24 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+-------------------------------------------------
+
+I believe that my iterative mergesort should be $O(n^2log(n))$
+
+the first thing it does is check if the array is either emtpy or of size 1 which is a constant +1
+
+This is because when it starts it enters two for loops based on the length of the array, which gives a complexity of $n*n$ or $n^2$
+
+After that it enters a while loop which runs until we can no longer split the array into halves, which is $log(n)$.  It also contains another for loop but as it is less than the $log(n)$ complexity it won't change it.
+
+so we have a complexity of $O(n*n\*log(n) + 1)$ = $O(n^2log(n))$
```

Comment:
This is different to the "normal" mergesort -- what causes the difference here?

Reply:
I apologize but I'm not sure I understand the question, which part you asking about? The halving, the for loop, or the overall complexity?